### PR TITLE
Upgrade transports compat tests to beta 7

### PR DESCRIPTION
**Pull request by weralabaj**

### CODE REVIEW

**Comment by ramonsmits** [comment]
@weralabaj  What was the reason that it previously didn't work?

File: src/TransportCompatibilityTests/AzureServiceBusV7/EndpointFacade.cs [hunk 1]
```diff
@@ -27,16 +25,15 @@ public class EndpointFacade : MarshalByRefObject, IEndpointFacade
...
                 .ConnectionString(AzureServiceBusConnectionStringBuilder.Build)
                 .Sanitization().UseStrategy<ValidateAndHashIfNeeded>();
 
             // TODO: remove when core v6 & asb V7 package are updated
             endpointConfiguration.UseSerialization<JsonSerializer>(); 
-            // Needed for callbacks package to work
-            endpointConfiguration.ScaleOut().InstanceDiscriminator(Guid.NewGuid().ToString("N"));
 
             endpointConfiguration.CustomConfigurationSource(new CustomConfiguration(endpointDefinition.Mappings));
+            endpointConfiguration.MakeInstanceUniquelyAddressable(Guid.NewGuid() + "N");
```
Comment (ramonsmits) [inline src/TransportCompatibilityTests/AzureServiceBusV7/EndpointFacade.cs]: Why not "A" here like for Rabbit and SQL and why the need to append the "N"?

Comment (weralabaj) [inline src/TransportCompatibilityTests/AzureServiceBusV7/EndpointFacade.cs]: Not sure, I just copied what was there before :P I'll change it.

**Comment by weralabaj** [comment]
> @weralabaj What was the reason that it previously didn't work?

Corrupted csproj file :P

**Pull request merged**